infra/terraform/test-org/github/labels.tf

```diff
@@ -15,9 +15,12 @@
  */
 
 locals {
+  remove_repo_labels = [
+  ]
+  sub_repos_labels = setsubtract(toset(local.repos), local.remove_repo_labels)
   repo_labels = {
     for o in flatten([
-      for repo in local.repos :
+      for repo in local.sub_repos_labels :
```

Comment:
Any reason not to use `local.repos` since remove_repo_labels is empty?

Reply:
Added `anthos-samples`